Assignment2/solrac1999-JSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+	"@context": {
+  	"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "prefix": "http://example.org/assignment2b/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "fullname": "rdfs:label",
+    "hometown": "rdfs:label",
+    "age": "rdfs:label",
+    "university":"rdfs:label",
+    "parents":"rdfs:label",
+    "mother":"rdfs:label",
+    "father":"rdfs:label",
+    "subjects":"rdfs:label",
+    "subject":"rdfs:label",
+    "teachers":"rdfs:label",
+    "teacher":"rdfs:label",

```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.